examples/cli/deploy.py

```diff
@@ -63,18 +69,31 @@ async def main(wallet_path, wallet_pw, nef_path, manifest_path, rpc_host):
             res = await client.get_version()
             account.sign_tx(tx, password=wallet_pw, magic=res.protocol.network)
             tx_id = await client.send_transaction(tx)
+            print(f"Contract hash: {get_contract_hash(tx.sender, nef.checksum, manifest.name)}")
             print(f"Transaction id: {tx_id}")
+            if poll_timeout is not None:
+                print("Polling for transaction status, please wait..")
+                status = await api.poll_tx_status(tx_id, client)
+                if status == vm.VMState.HALT:
+                    print(f"Transaction execution status: Success")
+                else:
+                    print(f"Transaction execution status: Failed (VMState: {status})")
+
         except api.JsonRpcError as e:
             print(e)
-            sys.exit(1)
+            sys.exit(-1)
+
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='Deploy contract')
     parser.add_argument("-w", metavar='WALLET', required=True, help="wallet.json path")
     parser.add_argument("-pw", metavar='PASSWORD', required=True, help="wallet password for signing")
     parser.add_argument("-c", metavar='CONTRACT_NEF', required=True, help="contract .NEF path")
     parser.add_argument("-m", metavar='CONTRACT_MANIFEST', required=True, help="contract .manifest.json path")
-    parser.add_argument("-s", metavar='RPC_SERVER', required=True, help="RPC server address e.g. https://mainnet1.neo.coz.io:443")
+    parser.add_argument("-s", metavar='RPC_SERVER', required=True,
+                        help="RPC server address e.g. https://mainnet1.neo.coz.io:443")
```

Comment:
I think we could set the testnet as default, or localhost. In general, I find it hard to find the correct endpoint and seed.

Reply:
Changing it to a different network will just shift the problem in time (until you need the next network). How about extending the help message with a link to https://dora.coz.io/monitor

Reply:
@ixje That is true, but at the same time, the developer will likely use testnet by default. Also, I would suggest using it to avoid sending 'real assets' in a test. If the user doesn't directly send it to the main net, it is because he may be doing a test. Also, Erik, it is worth mentioning something: First user experience.
What are the chances that they will use main net? I think it is pretty low. By setting localhost or testnet as default, we could increase user conversion, since they will be able to use it 'right away.

Thinking about it: To make this easier for the first use, we should include test wallets and neo-express configurations. With that, they will be able to run these scripts locally.
Does this make sense?

Reply:
Giving you some extra context:
I have a trauma that when I was working as product manager for Neo, they released the N3 version but it wasn't possible to run. You had to download it, change the configs, and then use it. It creates a bad experience. people want to be able to install it and run it in seconds, just to 'ensure it works'

Reply:
Just to clarify, the `RPC_SERVER` is not set by default, it's just part of the help message. You must always provide the argument yourself. I do agree we can provide a testnet server in the help message by default instead so they don't have to go to Dora to find one.  

I don't think we should provide test wallets + neo express configs because then we should also provide a manifest and test contract to make it complete. On one side I don't want todo that because we assume they run neo-express, while they could easily be running on regular test net. On the other side it's a maintenance burden to keep this all working and I'm not convinced yet we can carry that weight.

> Edit: Adding a place to get that configuration is super important. HOWEVER WE DON'T SHOW THE MAGIC ON DORA 😠 haha

what magic? the network magic? what do we need that for?